### ISSUE TITLE
(Android) Tap to hold feature

### ISSUE THREAD

**Issue by Reyestheknight** [yuzu-emu/yuzu#12168]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What feature are you suggesting?

A feature to change the button behavior from hold to press to tap to press. 

### Why would this feature be useful?

Certain games require holding the button, and this feature can make the touchscreen controls more flexible when playing the game. Super Mario Wonders is the best example where you have to hold the Y button to run and B to jump and you can't do both in Yuzu

**Comment by Hariu7** [comment]
Yes it required and another features should be introduced like combined button

**Comment by FrancescoDesogus** [comment]
Yes I agree, it would be very helpful to have
